src/util/std_expr.h

```diff
@@ -922,22 +914,31 @@ inline const minus_exprt &to_minus_expr(const exprt &expr)
 /// \copydoc to_minus_expr(const exprt &)
 inline minus_exprt &to_minus_expr(exprt &expr)
 {
-  PRECONDITION(expr.id()==ID_minus);
+  PRECONDITION(expr.id() == ID_minus);
   minus_exprt &ret = static_cast<minus_exprt &>(expr);
   validate_expr(ret);
   return ret;
 }
 
-
 /// \brief Binary multiplication
 /// Associativity is not specified.
-class mult_exprt:public multi_ary_exprt
+class mult_exprt : public multi_ary_exprt
 {
 public:
   mult_exprt(exprt _lhs, exprt _rhs)
     : multi_ary_exprt(std::move(_lhs), ID_mult, std::move(_rhs))
   {
   }
+
+  const exprt &lhs() const
```

Comment:
This is potentially a bit misleading as `mult_exprt` can have more than 2 operands.